File: 프로그래머스/0/120956. 옹알이 （1）/옹알이 （1）.py
def solution(babbling):
    answer = 0
    baby = ["aya", "ye", "woo", "ma"]
    for word in babbling:
        for b in baby:
            if b in word:
                word = word.replace(b,' ')
                if word.replace(' ', '') == '':
                    answer +=1
                    break
    return answer

File: 프로그래머스/0/120956. 옹알이 （1）/옹알이 （1）.java
class Solution {
    public int solution(String[] babbling) {
        int answer = 0;
        String[] baby = {"aya", "ye", "woo", "ma"};
        for (String word : babbling) {
            for (String b : baby) {
                if (word.contains(b)) {
                    word = word.replace(b, " ");
                    if (word.replace(" ", "").equals("")) {
                        answer += 1;
                        break;
                    }
                }
            }
        }
        return answer;
    }
}